### ISSUE TITLE
prosemirror-view drag selection delay breaks table selections in Safari

### ISSUE THREAD

**Issue by luke-john-atlassian** [ProseMirror/prosemirror#1234]
**Context:**

prosemirror-view@1.19.2 (via https://github.com/ProseMirror/prosemirror-view/commit/885258b80551ac87b81601d3ed25f552aeb22293) introduced a drag selection sync delay to avoid some Chrome specific bugs https://github.com/ProseMirror/prosemirror/issues/1152.

This change breaks https://github.com/ProseMirror/prosemirror-tables cell selections in Safari.

Note: our team does not use the open source prosemirror-tables plugin, but has a similar (forked) plugin which also broke in a similar way, we have currently worked around this by "opting out" of the drag selection delay in Safari.

**Reproduction link:**

https://codesandbox.io/s/headless-https-nle9e

**Steps to reproduce:**

click and hold in lower right cell to start a selection

continue holding and move cursor to cell above

**Actual behaviour**

Cell selection is created containing the two cells, and no text in other cells is selected.

Safari screen shot
![Screen Shot 2022-01-13 at 8 58 40 am](https://user-images.githubusercontent.com/87837553/149246973-63c78133-8a0a-4243-9ed4-66c190fcb257.png)

**Expected behaviour**

Cell selection is created containing the two cells, and text in other cells is presented as selected.

note: pressing ctrl+c results in only the cell selection being added to the clipboard. 

Chrome screen shot
![Screen Shot 2022-01-13 at 9 00 57 am](https://user-images.githubusercontent.com/87837553/149247150-3a45ce29-b1d0-4f2f-a79c-1940bedb56ef.png)

**Comment by marijnh** [comment]
Does adding an `!browser.chrome` to the `if` that sets up the delayed sync solve this for you? That seems like the least impactful solution.

**Comment by luke-john-atlassian** [comment]
It does solve it yes (though it's `browser.chrome` rather than `!browser.chrome`).  This is roughly what our workaround does (though we limited it to Safari, and instead prevented that conditional running by setting view.mouseDown to null (and manually cleaning up old event listeners when mouse up events aren't received)).

I've opened a draft PR against `prosemirror-view` with that change https://github.com/ProseMirror/prosemirror-view/pull/116.
